lib/auth/apiserver.go

```diff
@@ -96,6 +96,7 @@ func NewAPIServer(config *APIConfig) (http.Handler, error) {
 	// Operations on certificate authorities
 	srv.GET("/:version/domain", srv.withAuth(srv.getDomainName))
 	srv.GET("/:version/cacert", srv.withAuth(srv.getClusterCACert))
+	srv.GET("/:version/cacerts", srv.withAuth(srv.getClusterCACerts))
```

Comment:
Sorry I have only just realized now that you're adding this method to the HTTP API which is deprecated. All new API methods should be added to GRPC server IIRC. @Joerger Is this correct?

Reply:
Yeah, let's add this to the gRPC service and make it either (1) return all CAs of all types or (2) return CAs of one type specified as a request parameter. Don't make it return `HostCA` only, implicitly.

Reply:
I just switched this to a new gRPC method but just saw your comment now @awly about potentially returning other CA certs. Do you have a preference between returning all certs or requiring the parameter? Both are doable